react/src/components/user/course/CourseHero.jsx

```
@@ -0,0 +1,184 @@
+                <div className="price">
+                  {course.discount_price ? (
+                    <>
+                      <span className="original-price">${parseFloat(course.price).toFixed(2)}</span>
+                      <span className="current-price">${parseFloat(course.discount_price).toFixed(2)}</span>
+                    </>
+                  ) : (
+                    <span className="current-price">${parseFloat(course.price).toFixed(2)}</span>
+                  )}
+                </div>
+                
+                <button className="btn btn-enroll w-100 mb-3">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)